lambdalib/padring/README.md

```diff
@@ -0,0 +1,98 @@
+# PADRING GENERATOR
+
+## Introduction
+
+The lamdbdalib `padring` library is an automated "pure verilog" padring generator with support for cells within the [IOLIB](../../iolib/README.md) io cell library.
+
+## PARAMETERS
+
+### {NO,EA,WE,SO}NCELLS
+Specifies the total number of placed cells within one side of the padring, includig supply and clamp cells.
+
+### {NO,EA,WE,SO}NPINS
+Specifies the total number of logical device pins (pads) connected to one side of the padring, not including supply pins. The `CELLMAP` parameter specifies which one of the pins should be connected to a cell.
+
+### {NO,EA,WE,SO}NSECTIONS
+Specifies the total number of power sections within one side of the padring. The `CELLMAP` parameter specifies which one of the power sections should be connected to a cell.
+
+### {NO,EA,WE,SO}CELLMAP
+Specifies the type of cells, pin connections, properties, and power connections of all cells in the padring with the exception of filler cells. The physical placement of the cells within the padring shall be done in the order dictated by `CELLMAP`. The CELLMAP is a vector of size NCELLS * 40, with the 40bit vector split into the following fields:
+
+  * PIN[7:0]     = CELLMAP[7:0]   = pin number connected to cell
+  * COMP[7:0]    = CELLMAP[15:8]  = pin number for complementary pad for differential cells
```

Comment:
do we care which one is positive? if so we should indicate that PIN should be + and COMP should be -

Reply:
done